infra/modules/service/main.tf

```
@@ -52,9 +60,37 @@ resource "aws_lb" "alb" {
   # Note that header names cannot contain underscores
   # https://docs.bridgecrew.io/docs/ensure-that-alb-drops-http-headers
   drop_invalid_header_fields = true
-
-  # TODO(https://github.com/navapbc/template-infra/issues/162) Add access logs
   # checkov:skip=CKV_AWS_91:Add access logs in future PR
+  access_logs {
+    bucket  = aws_s3_bucket.load_balancer_logs.id
+    prefix  = "${var.service_name}-lb"
+    enabled = true
+  }
+}
+
+resource "aws_s3_bucket" "load_balancer_logs" {
+  bucket = "${var.service_name}-access-logs${var.s3_suffix}"
+}
+
+resource "aws_s3_bucket_policy" "log_access_bucket_policy" {
```

nit: since the resource type is aws_s3_bucket_policy, including bucket_policy in the name is redundant. also, in this case i also think we can be a tiny bit more specific about the permissions i.e. rather than it providing access to logs, it's about providing put access to the "load balancer logs" bucket. here's a suggestion

```suggestion
resource "aws_s3_bucket_policy" "load_balancer_logs_put_access" {
```